mne/forward/_field_interpolation.py

```diff
@@ -279,7 +279,7 @@ def _make_surface_mapping(info, surf, ch_type='meg', trans=None, mode='fast',
         logger.info('Prepare EEG mapping...')
     if len(picks) == 0:
         raise RuntimeError('cannot map, no channels found')
-    chs = pick_info(info, picks, copy=True)['chs']
```

Comment:
Oops, this is cruft from earlier work where I was going to remove it from `pick_info` too but decided against it